devo/api/client.py

```diff
@@ -357,8 +379,7 @@ def query(self, query=None, query_id=None, dates=None,
                 toDate = self._toDate_parser(fromDate,default_to(dates['to']))
 
                 if toDate > default_to("now()"):
-                   return DevoClientException("Modes 'xls' and 'msgpack' does not"
-                                f" support future queries because KeepAlive tokens are not available for those resonses type")
+                   raise raise_exception("Modes 'xls' and 'msgpack' does not support future queries because KeepAlive tokens are not available for those resonses type",400)
```

Comment:
If we don't have address, we cannot create an HTTP request and no HTTP status code can be returned. It will create confussion in developer/use as no HTTP service can have returned any status code. Do not create HTTP status codes. Just returns the one returned by Serrea.